drakrun/drakrun/draksetup.py

```diff
@@ -245,6 +246,79 @@ def send_usage_report(report):
         logging.exception("Failed to send usage report. This is not a serious problem.")
 
 
+def create_rekall_profiles(install_info, pdb_guid):
+    profiles_path = os.path.join(LIB_DIR, "profiles")
+    mount_path = ''.join(random.choice(string.ascii_letters + string.digits) for i in range(20))
+    os.mkdir(mount_path)
+
+    '''
+    Note: we mount 2nd partition, as 1st partition is windows boot related
+    and 2nd partition is C:\\
+    '''
+
+    if install_info["storage_backend"] == "zfs":
+        # workaround for not being able to mount a snapshot
+        base_snap = shlex.quote(os.path.join(install_info["zfs_tank_name"], 'vm-0@booted'))
+        tmp_snap = shlex.quote(os.path.join(install_info["zfs_tank_name"], 'tmp'))
+        subprocess.check_output(f'zfs clone {base_snap} {tmp_snap}', shell=True)
+
+        tmp_mount = shlex.quote(os.path.join("/dev/zvol", install_info["zfs_tank_name"], "tmp-part2"))
+        subprocess.check_output(f'mount -t ntfs -o ro {tmp_mount} {mount_path}')
+    else:  # qcow2
+        subprocess.check_output("modprobe nbd")
+        # TODO: this assumes /dev/nbd0 is free
+        subprocess.check_output(' '.join([
+            "qemu-nbd",
+            "-c",
+            "/dev/nbd0",
+            "-P",
+            "2"
+            "--read-only",
+            os.path.join(LIB_DIR, "volumes", "vm-0.img")
+        ]), shell=True)
```

Comment:
Does it work? Seems like we're not mounting the `nbd0` device anywhere :thinking: